librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -4,4 +4,47 @@ commands:
   - name: dc
     cname: debug_continue_oldhandler
     summary: Continue execution
-    type: RZ_CMD_DESC_TYPE_OLDINPUT
\ No newline at end of file
+    type: RZ_CMD_DESC_TYPE_OLDINPUT
+  - name: ds
+    summary: Debug step commands
+    subcommands:
+      - name: dsu
+        summary: Debug step until commands
+        subcommands:
+          - name: dsu
+            cname: cmd_debug_step_until
+            summary: Step until <address>
+            args:
+            - name: addr
+              type: RZ_CMD_ARG_TYPE_RZNUM
+              flags: RZ_CMD_ARG_FLAG_LAST
```

Comment:
no need for this, i've changed cmd_descs_generate.py to automatically set it when appropriate.

Reply:
You were referring to  just the `flags: ` descriptor, right?

Reply:
yes, just `flags`.